### PR TITLE
Header

### PR DESCRIPTION
**Pull request by oliferwolox**
## Summary

- Add header with customBackButton

## Screenshots

header-notch

## Trello Card

https://trello.com/c/RhjYf8oM/15-header

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,21 +1,50 @@
...
 import { NavigationContainer } from '@react-navigation/native';
 import { createStackNavigator } from '@react-navigation/stack';
 
 import LibraryScreen from '@screens/LibraryScreen';
 import BookDetailScreen from '@screens/BookDetailScreen';
+import CustomHeader from '@components/CustomHeader';
 import { ROUTES } from '@constants/routes';
+import { COLORS } from '@constants/colors';
+
+const backButton = require('@assets/Navigationbar/ic_back.png');
```
Comment (IgnacioAbadie) [inline App.js:13]: import assets like other imports

File: App.js [hunk 2]
```diff
@@ -1,21 +1,50 @@
...
+          gestureEnabled: true,
+          headerStyle: {
+            height: 80
+          },
+          headerTintColor: COLORS.white,
+          headerTitleStyle: {
+            textTransform: 'uppercase'
+          },
+          headerBackImage: () => (
+            <Image source={backButton} style={{ width: 30, height: 30 }} />
```
Comment (IgnacioAbadie) [inline App.js:34]: create s file for styles or a new styles constant in this file. the use the styles on Image components, header style, titleStyle, etc..

File: src/constants/routes.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
 export const ROUTES = {
   Library: 'Library',
-  BookDetail: 'BookDetail'
+  BookDetail: 'Book Detail'
```
Comment (IgnacioAbadie) [inline src/constants/routes.js:3]: let UpperCamelCase for route names. If you want to set a title for header pass a title string on Stack.screen options
https://reactnavigation.org/docs/headers/#setting-the-header-title

**Comment by IgnacioAbadie** [comment]
@oliferwolox Please take care about your commits. You are submitting english/spanish commits and check the lint before commit command `yarn run lint`

File: App.js [hunk 4]
```diff
@@ -1,21 +1,50 @@
...
+          ),
+          header: props => <CustomHeader {...props} />,
```
Comment (marianozicavo) [inline App.js:36]: Try if this works the same

```suggestion
          header: CustomHeader,
```

**Comment by oliferwolox** [comment]
> Nice work! But the title of the header isn't centered like in Trello

@marianozicavo  But its Android, i think the IOS view is centered.

**Pull request merged**